R/AbstractGraphReporter.R

```diff
@@ -482,10 +482,6 @@ AbstractGraphReporter <- R6::R6Class(
                 }
 
                 # Add legend
-                # but it is broken if there is only one level
-                # so hard-code for now to skip if there is only one level
-                # TODO: remove if statement when the following issue is resolved:
-                # https://github.com/datastorm-open/visNetwork/issues/290
```

Comment:
@jayqi I created #177, feel free to edit it if you think I didn't capture the intent of this